blog/_posts/ride-the-lightning-btcpay.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Ride The Lightning: An open-source project making node management easy for everyone"
+author: pavlanex
```

Comment:
```suggestion
author: Pavlenex
```

Reply:
Thanks for pointing out these typos, @Zaxounette! I think we'll keep it lowercase to be consistent with pavlenex's preferred pseudonym styling, but I've now corrected the a/e typo in a subsequent commit.